Source/MBF.Displays.OLED1Xplained.pas

```diff
@@ -0,0 +1,704 @@
+unit MBF.Displays.OLED1Xplained;
+{
```

Comment:
Also, as written in E-Mail we should think about two kinds of Display Units, one with Framebuffer, one without...

Reply:
A screenbuffer could just be a single pointer.
That is assigned or not.
And used accordingly.

Reply:
One problem of pxl for microcontrollers was that it is very well designed, which caused huge impacts on performance and filesize. 
When we allow both framebuffer/NoFramebuffer in one object my fear is that we increase the codesize a lot by always having framebuffer/non-framebuffer code in one procedure/function.

Whith often only 8 or 16k flash this hurts, when you have seperate implementations the design is compromised but you may be able to run the code on devices with lower specs.

I am not sure if I have written that before, I am dtruggeling with myself here as there is a rich choice of boards with plenty of ram/flash but usually Jeppe and I got requests to create header files for boards with crazy low ram/flash footprint.

Reply:
Again, you are right.
I will think about it.
Perhaps, a framebuffer object can be injected into a display object abstraction. Too late now. Will think with fresh brains tomorrow !